### DISCUSSION TITLE
Enter URL of this repo to "URL for extension's git repository"

### DISCUSSION THREAD

**Discussion by MounseyWood** [Mikubill/sd-webui-controlnet#648] [Q&A] [answered]
Hey, I'm a newbie, and struggling with installing this. I'm in Microsoft Edge, I've opened up the extensions in developer mode. It gives me an option to 'load unpacked', then I have to select a file/folder. If I select the files I've downloaded I get the error message 'manifest file is missing or unreadable'. There doesn't appear to be an option to enter a URL in Edge. Where could I be going wrong? Many thanks, Matthew

**Comment by MounseyWood** [comment, marked as answer]
It's amazing what a night's sleep can do. I found the correct 'extensions' tab... ><

**Reply by masonmacphail** [reply]
@MounseyWood , where is the correct tab? I'm having the same problem.

**Reply by masonmacphail** [reply]
Welp, would have helped if I had downloaded stable diffusion first. I see the extension tab within stable diffusion

**Reply by hritik1a** [reply]
Can't understand this , can you help. I've installed stable diffusion
